portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx

```diff
@@ -198,6 +198,12 @@ function Endpoints(props) {
                     }
                     return { ...initState, endpointConfig: config, endpointImplementationType: 'ENDPOINT' };
                 }
+                if (endpointType === 'INLINE') {
```

Comment:
The codebase intends to branch on endpointImplementationType. Adding a logic to handle behavior specifically for the INLINE type would duplicate that logic and could lead to inconsistencies if the two fields ever differ. Let’s avoid adding this special case here.